client/components/ui/date-picker.js

```diff
@@ -24,8 +28,66 @@ export default class DatePicker extends React.Component {
         });
     }
 
+    setMaxOrMin(pickerOptions, props) {
+        // Maybe a minimum or maximum value is set
+        pickerOptions.max = props.maxDate ? new Date(props.maxDate) : false;
+        pickerOptions.min = props.minDate ? new Date(props.minDate) : false;
+        return pickerOptions;
+    }
+
+    componentWillReceiveProps(newProps) {
+        let pickerOptions = this.setMaxOrMin({}, newProps);
+        this.pickadate.set(pickerOptions);
+    }
+
+    localizationTable(prefix, tableToLocalize) {
+        let pref = prefix.length ? `client.datepicker.${prefix}.` : '';
```

Comment:
Actually `prefix` is always of length > 0, so this conditional could be removed.